### ISSUE TITLE
ETL Cooling Plate Section

### ISSUE THREAD

**Issue by cfangmeier** [nebraska-silicon-lab/CAD#5]

**Comment by cfangmeier** [comment]
![IMG_20200824_153501](https://user-images.githubusercontent.com/2569566/91332341-a7d13000-e791-11ea-83a2-6b50eaf43372.jpg)

The plate and tube have been fabricated and holes drilled to support single-side mounting of modules. The modules that will be mounted on the reverse-side will be shifted 50% of their width and will require additional mounting holes.

**Comment by fgolf** [comment]
We want the holes on the other (let's call it front) side to be a mirror image of what is here, i.e. the holes shown (back side) have a row of modules along the top and bottom with a service hybrid in between.  The holes on the other (front) side would mount a row of modules shadowing the service hybrid on the back side and rows of service hybrids shadowing the modules on the back side.

**Comment by cfangmeier** [comment]
The Plates were glued together today. Below image shows the glue application

![image](https://user-images.githubusercontent.com/2569566/101409781-fbc8d600-38a3-11eb-905e-10af541c54dd.png)

Keith notes:

> There are a few small areas with no glue dots. I used a q-tip and added some small dots by hand in those areas to hopefully have even coverage.

After applying the glue, Keith filled in any remaining voids around the pipe with thermal paste before placing the top piece and weighing the whole thing down.

![PXL_20201207_202846913](https://user-images.githubusercontent.com/2569566/101409965-434f6200-38a4-11eb-90ea-253e8844b5b2.jpg)

**Comment by cfangmeier** [comment]
I retrieved the plate from the shop. Shown below is the plate staged with 1 service hybrid and 12 modules, of which 4 currently have heaters and only one has the heater glued on.

![PXL_20201210_181714034](https://user-images.githubusercontent.com/2569566/101814941-6bc59f00-3ae4-11eb-9ebb-2c196ad109e3.jpg)

One issue I discovered with the service-hybrid is that when I adjusted the design to use some better connectors I accidentally removed the mounting holes (Dang EDA software!). To compensate for this, we can drill holes through the PCBs ourselves, but will need to use non-conductive screws to avoid shorting out the board. Something in the ballpark of [1] should work. I don't think this will dramatically reduce the usefulness of this mockup because (1) we are more concerned with the modules than the service hybrid and (2) most of the cooling is happening through the PCB-disc contact area, not through the mounting screws.

[1] https://www.mcmaster.com/screws/head-type~socket/material~plastic/
